src/assets/scripts/client/navigationLibrary/StandardRoute/StandardRouteWaypointModel.js

```diff
@@ -377,25 +372,25 @@ export default class StandardRouteWaypointModel extends BaseModel {
      * - "S210"
      *
      * @for StandardRouteWaypointModel
-     * @method _parseWaypointRestrictions
+     * @method _setRestrictions
      * @param waypointRestrictions {string}
      * @private
      */
-    _parseWaypointRestrictions(waypointRestrictions) {
+    _setRestrictions(waypointRestrictions) {
```

Comment:
😬 

I've always been taught that the word `set` has a very particular meaning in programming.  I can't tell you how many times I used the word and was scolded for using it incorrectly.  In this case, I'm not sure `set` is the correct term to use here.  Are we actually setting a value in this method?  Or are we parsing the data and then _calling another method_ that sets the data?

The classical definition, as I was taught:
**Get**: Should return a property value and cause no side effects
**Set**: Should accept a single value and set a single property and cause no side effects